acceptance/acceptance_test.go

```diff
@@ -739,27 +739,14 @@ func testAcceptance(
 						assertMockAppRunsWithOutput(t, repoName, "Launch Dep Contents", "Cached Dep Contents")
 
 						t.Log("restores the cache")
-						if creatorSupported {
-							h.AssertContainsMatch(t, output, `(?i)\[creator] Restoring data for "simple/layers:cached-launch-layer" from cache`)
-							h.AssertContainsMatch(t, output, `(?i)\[creator] Restoring metadata for "simple/layers:cached-launch-layer" from app image`)
-						} else {
-							h.AssertContainsMatch(t, output, `(?i)\[restorer] Restoring data for "simple/layers:cached-launch-layer" from cache`)
-							h.AssertContainsMatch(t, output, `(?i)\[analyzer] Restoring metadata for "simple/layers:cached-launch-layer" from app image`)
-						}
+						h.AssertContainsMatch(t, output, `(?i)Restoring data for "simple/layers:cached-launch-layer" from cache`)
```

Comment:
I love this as it's much cleaner. I worry that we're losing the ability to assert that the creator (vs. not creator) was actually run. This is unit tested pretty well, so maybe it's okay. But for example the assertions here, I wouldn't want to change: https://github.com/buildpacks/pack/pull/644/files#diff-38e2f40dfce97bbc5beadcb903ffe700R693-R696.

Reply:
Yeah, I went through the same thought process but as you noted, we already have a specific acceptance set for that.